pkg/gatherers/conditional/gathering_rule.schema.json

```diff
@@ -104,6 +104,12 @@
                             "title": "TailLines",
                             "minimum": 1,
                             "maximum": 4096
+                        },
+                        "max_containers": {
```

Comment:
One of the initial purposes of the task was that there would be a hard limit on the operator side so that when we define the conditions externally, it can't collect too much.

Reply:
How about now?